packages/orchestration/src/util/filtering.ts

```diff
@@ -58,3 +60,17 @@ export function buildAzureContentSafetyFilter(
     })
   };
 }
+
+/**
+ * Convenience function to create Azure content filters.
+ * @param config - Configuration for Llama guard filter.
+ * @returns Filter config object.
+ */
+export function buildLlamaGuardFilter(
+  config?: LlamaGuard38B
```

Comment:
[req] config should be a required property.

Reply:
My idea was to apply default if nothing is provided by the user.
Orchestration service does not allow an empty config `{}` object to be sent. So I applied a default of `false` for all categories.

Reply:
No longer relevant.